src/Farmer/Arm/Compute.fs

```diff
@@ -7,7 +7,7 @@ open Farmer.Vm
 open System
 open System.Text
 
-let virtualMachines = ResourceType ("Microsoft.Compute/virtualMachines", "2018-10-01")
+let virtualMachines = ResourceType ("Microsoft.Compute/virtualMachines", "2019-03-01")
```

Comment:
Version 2019-03-01 is the oldest version to support the spot instance properties.